miden-lib/src/notes/utils.rs

```diff
@@ -15,21 +17,16 @@ pub fn build_note_script(bytes: &[u8]) -> Result<NoteScript, NoteError> {
     Ok(note_script)
 }
 
-/// Creates the RECIPIENT for the P2ID note script created by the SWAP script
-pub fn build_p2id_recipient(target: AccountId, serial_num: Word) -> Result<Digest, NoteError> {
+/// Creates a [NoteRecipient] for the P2ID note script created by the SWAP script.
```

Comment:
nit: Is this comment right? It seems that this would work generically for any P2ID-related usecase, not only SWAP. Doesn't seem important but since it's a public function it might be confusing.